postinstall.one

```diff
@@ -32,7 +32,7 @@ then
 fi
 
 if [ -f /etc/init.d/one-context ]; then
-    if [ -d /etc/sysconfig/network-scripts ]; then
+    if [ -d /etc/sysconfig/network-scripts ] || [ -d /etc/sysconfig/network ]; then
```

Comment:
I used [this](https://github.com/OpenNebula/addon-context-linux/releases/download/v5.0.3/one-context_5.0.3.rpm) rpm package.
It have no any systemd units inside, so `systemctl enable vmcontext` is not working, because `systemd-sysv-generator` is not work propertly.
But `chkconfig --set vmcontext on` is works fine, that's why I made this change.

Right, we are provided [new systemd units](https://github.com/OpenNebula/addon-context-linux/blob/master/src/usr/lib/systemd/system/) for now, so I think it will work fine with it.

But why [this lines](https://github.com/OpenNebula/addon-context-linux/blob/f5c28e896ca3239c644d77ed7772b4527940d55c/postinstall.one#L35-L37) is needed? - Centos7 is uses systemd too